autopts/wid/mcp.py

```diff
@@ -463,15 +463,50 @@ def hdl_wid_20136(params: WIDParams):
 def hdl_wid_20206(params: WIDParams):
     """Please verify that for each supported characteristic, attribute handle/UUID
      pair(s) is returned to the upper tester.
-     Mute: Attribute Handle = 0x00A2
-     Characteristic Properties = 0x1A
-     Handle = 0x00A3
-     UUID = 0x2BC3
+     Media Player Name: Attribute Handle = 0x00E2
+     Characteristic Properties = 0x12
+     Handle = 0x00E3
+     UUID = 0x2B93
+
+     Media Player Icon Object: Attribute Handle = 0x00E5
+     Characteristic Properties = 0x02
+     Handle = 0x00E6
+     UUID = 0x2B94
+     ...
      """
 
-    # There is no way to do that with current API.
+    stack = get_stack()
 
-    return True
+    if params.test_case_name == "MCP/CL/CGGIT/SER/BV-02-C":
+        chars = (stack.mcp.event_queues[defs.MCP_DISCOVERED_EV][0][3:25])
+        chrc_list = ['{:04X}'.format(chrc).upper() for chrc in chars]
```

Comment:
Can be simplified, big X gives upper case already.
```suggestion
        chrc_list = [f'{chrc:04X}' for chrc in chars]
```